packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/postprocessor/implementation/CodeFormatterUtil.java

```diff
@@ -36,9 +45,7 @@ public final class CodeFormatterUtil {
      * @param plugin The plugin to use to write the formatted files.
      */
     public static void formatCode(Map<String, String> files, NewPlugin plugin, Logger logger) {
-        for (Map.Entry<String, String> file : formatCodeInternal(files.entrySet(), logger)) {
-            plugin.writeFile(file.getKey(), file.getValue(), null);
-        }
+        formatCodeInternal(files, logger).forEach(entry -> plugin.writeFile(entry.getKey(), entry.getValue(), null));
```

Comment:
I'm sorry I don’t quite recall the reasons for removing Eclipse JDT, besides the larger shipped JAR size. Just trying to understand if there are any potential issues with using Eclipse JDT?

Reply:
It was two things:

1. Shipping size used to be a larger concern, as at the time Eclipse JDT Code in-memory code formatting was removed last time the shipped JAR size was 50-100MB all up. This is no longer the case, it's closer to 20MB now.
2. I realize we had some version skew, so our Spotless run in azure-sdk-for-java produced different code formatting than the generator did, causing noise and CI failures when validating code generation in CI. Now both Spotless and the code formatting here should be equivalent.